cli/jobs/pipelines/automl/cli-automl-classification-task-bankmarketing-pipeline/components/src_register/register.py

```diff
@@ -0,0 +1,77 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+import argparse
+import json
+import os
+import time
+
+
+from azureml.core import Run
+
+import mlflow
+import mlflow.sklearn
+
+# Based on example:
+# https://docs.microsoft.com/en-us/azure/machine-learning/how-to-train-cli
+# which references
+# https://github.com/Azure/azureml-examples/tree/main/cli/jobs/train/lightgbm/iris
+
+
+def parse_args():
+    # setup arg parser
+    parser = argparse.ArgumentParser()
+
+    # add arguments
+    parser.add_argument("--model_input_path", type=str, help="Path to input model")
+    parser.add_argument(
+        "--model_base_name", type=str, help="Name of the registered model"
+    )
+
+    # parse args
+    args = parser.parse_args()
+    print("Path: "+ args.model_input_path)
+    # return args
+    return args
+
+
+def main(args):
+    '''
+    Register Model Example
+    '''
+    #Set Tracking URI
+    current_experiment = Run.get_context().experiment
+    tracking_uri = current_experiment.workspace.get_mlflow_tracking_uri()
+    print("tracking_uri: {0}".format(tracking_uri))
+    mlflow.set_tracking_uri(tracking_uri)
+    mlflow.set_experiment(current_experiment.name)
+
+    #Get Run ID from model path
+    print("Getting model path")
+    mlmodel_path = os.path.join(args.model_input_path, "MLmodel")
+    runid = ""
+    with open(mlmodel_path, "r") as modelfile:
+        for line in modelfile:
+            if "run_id" in line:
+                runid = line.split(":")[1].strip()
```

Comment:
Could we get the run id from mlflow api?

Reply:
Raised a new bug for this https://msdata.visualstudio.com/Vienna/_workitems/edit/1969749 as classification job returns actual mount path instead of URI path which can not be consumed directly.

Reply:
@vikasagrawal-ms from what I understand, this is expected. Best model (in mlflow structure) is passed over as input to next node, which will be a mount path.

Reply:
In this case we need a URI to register the model, a mount path is not accepted . For ease of compatibility with other components an option can be added for URI instead of mount path

Reply:
@wangchao1230 Is it expected for the model output of a component to be a path and not a URI? Is there a simpler way to obtain URI path to the model instead of the mount path?